apps/webapp/package.json

```
@@ -12,6 +12,7 @@
     "format": "prettier --write .",
     "lint": "eslint --cache --cache-location ./node_modules/.cache/eslint .",
     "start": "cross-env NODE_ENV=production node --max-old-space-size=8192 ./build/server.js",
+    "start-from-docker": "cross-env node --max-old-space-size=8192 ./build/server.js",
```

could it be `start:local`, maybe?
```suggestion
    "start:local": "cross-env node --max-old-space-size=8192 ./build/server.js",
```

---

The reason for avoiding `start:local` is to avoid the misunderstanding that they are started together by the `run-s start:*` command, like `dev` and `dev:*`.
However, the name really could be anything, so I will modify it to `start:local`.

---

If you want, you can use something like `docker:start` or something like that. I commented because except for `build-storybook`, we don't have other script commands that use `-`. Feel free to use another name like `docker:start` or something like that.

Thanks for accepting my review 😄